analyzer-comments/java/annalyns-infiltration/avoid_comparison_with_literal.md

```diff
@@ -0,0 +1,11 @@
+# Avoid comparison with boolean literals
+
+Instead of comparing booleans with the literal values, it can be simplified to just the variables: 
+
+```java
+// instead of
+return knightIsAwake == true
+
+// use
+return knightIsAwake
```

Comment:
Let's make these two expressions valid Java, so end them with a semicolon:

```suggestion
// instead of
return knightIsAwake == true;

// use
return knightIsAwake;
```